python/.python-version

```diff
@@ -1,3 +1,3 @@
 3.7.12
 3.8.12
-3.9.10
+3.9.9
```

Comment:
Actually I probably should not commit this change, but when I do the listing of python 3.9 in pyenv, somehow I cannot find the 3.9.10

```
(venv) ➜  python git:(master) ✗ pyenv install --list | grep 3.9
  3.9.0
  3.9-dev
  3.9.1
  3.9.2
  3.9.4
  3.9.5
  3.9.6
  3.9.7
  3.9.8
  3.9.9
  miniconda-3.9.1
  miniconda3-3.9.1
  miniconda3-3.9-4.9.2
  miniconda3-3.9-4.10.3
```

Reply:
Does `tox --recreate` help by any chance?

Reply:
Nope, my pyenv is installed outside tox so it might be related, but I can exclude this file from PR and keep it locally to test for python 3.9